applications/asset_tracker_v2/src/ext_sensors/ext_sensors.c

```diff
@@ -0,0 +1,269 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <string.h>
+#include <drivers/sensor.h>
+#include "ext_sensors.h"
+#include <stdlib.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(ext_sensors, CONFIG_EXTERNAL_SENSORS_LOG_LEVEL);
+
+/* Convert to s/m2 depending on the maximum measured range used for adxl362. */
+#if defined(CONFIG_ADXL362_ACCEL_RANGE_2G)
+#define ADXL362_RANGE_MAX_M_S2 19.6133
+#elif defined(CONFIG_ADXL362_ACCEL_RANGE_4G)
+#define ADXL362_RANGE_MAX_M_S2 39.2266
+#elif defined(CONFIG_ADXL362_ACCEL_RANGE_8G)
+#define ADXL362_RANGE_MAX_M_S2 78.4532
+#endif
+
+#define ADXL362_THRESHOLD_RESOLUTION_DECIMAL_MAX 2048
+
+/* Local accelerometer threshold value. Used to filter out unwanted values in
+ * the callback from the accelerometer.
+ */
+double threshold = ADXL362_RANGE_MAX_M_S2;
+
+struct env_sensor {
+	enum sensor_channel channel;
+	uint8_t *dev_name;
```

Comment:
Making this `const char*` would be clearer, no?

Reply:
Agreed, fixed